File: IO/Input/input1_CLIargs.go
// 1. Done

package main

import (
	"fmt"
	"os"
	"strconv"
)

var ()

func main() {
	name := os.Args[1]                            // Note [0] would be filename
	age, _ := strconv.ParseInt(os.Args[2], 10, 0) // No error handling
	fmt.Printf("Go CLI arg name : %s, and age : %d\n", name, 12*age)
}

File: IO/Input/input1_CLIargs.js
// 1. Done.

"use strict"; /* jshint
node : true
*/

var args = process.argv.slice(2); // Discard 'node' and 'filename' from the array of arguments
process.stdout.write("CLI args name is " + args[0].toUpperCase()); // Trialling a more direct access to stdout than console.log
process.stdout.write(", and age in months is " + args[1] * 12 + "\n");
